### DISCUSSION TITLE
Hide child comments when parent comment is hidden.

### DISCUSSION THREAD

**Discussion by TimothyBJacobs** [community/community#3609] [Discussions]
In a discussion, if a parent comment in the thread is hidden each child comment is still displayed.

![Screen Shot 2021-03-25 at 1 54 28 PM](https://user-images.githubusercontent.com/3460448/112520372-b7d5e000-8d71-11eb-99df-e1248e13dbfd.png)

**Comment by gotom22** [comment]
I second that. Very tedious to "clean up" if every child comment has to be hidden individually. Kind of defeats the purpose.

**Comment by lunaleaps** [comment]
Yes this is an issue for our public discussion where we coordinate releases and once a top-level comment is resolved, ideally all follow-up should collapse

**Reply by cortinico** [reply]
Seconding @lunaleaps here. The link to our discussion is here: https://github.com/reactwg/react-native-releases/discussions/13 to give you a sense of what's a real use case of this.
